components/feature/addons/src/main/java/mozilla/components/feature/addons/update/AddonUpdater.kt

```diff
@@ -186,8 +186,9 @@ class DefaultAddonUpdater(
     ) {
         logger.info("onUpdatePermissionRequest $current")
 
+        val shouldGrantWithoutPrompt = Addon.localizePermissions(newPermissions).isEmpty()
         val wasPreviouslyAllowed =
```

Comment:
nit: would rename this val now. It's no longer just "was previously allowed", maybe "shouldShowNotification" or "shouldShowPrompt"?